runbot_website_display/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+======================
+Runbot display options
+======================
+
+This module was written to allow users to reorder runbot branches and to hide 
```

Comment:
@hbrunn Thanks for the contribution
"This module allows users to reorder runbot branches and to hide uninteresting ones."